### PR TITLE
Addresses #2015 Update definitions for bladder urothelial cell types

### PR DESCRIPTION
**Pull request by bvarner-ebi**
Addresses #2015 Update definitions for bladder urothelial cell types

### CODE REVIEW

**Comment by bvarner-ebi** [comment]
#gogoeditdiff

**Comment by dosumis** [comment]
Please see comments on ticket.

**Comment by bvarner-ebi** [comment]
#gogoeditdiff

**Review by gouttegd — CHANGES_REQUESTED** [review]
Everything looks fine for CL:1000442.

Incorrect annotation property (`rdfs:comment` instead of `IAO:0000115`) and typo for CL:1001428.

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -27608,11 +27607,9 @@ SubClassOf(obo:CL_1001320 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000005
 # Class: obo:CL_1001428 (bladder urothelial cell)
 
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_1001428 "KUPO:0001121")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1038/s41385-022-00565-0") rdfs:comment obo:CL_1001428 "A urothellial cell that is part of the urothelium of the urinary bladder.")
```
[the remark this answers is not in the document]
Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:27610]: Thank you for spotting both.
I've noticed Protégé will sometimes default to rdfs:comment. Thank you for catching.

[the remark this answers is not in the document]
Comment (gouttegd) [reply src/ontology/cl-edit.owl:27610]: And coming back to an earlier discussion: In this case, should it be “A**n** urothelial cell” or “A urothelial cell”? (Genuine question; I’d be inclined to say “an”, but I’ll happily yield to native English speakers on that point.)

[the remark this answers is not in the document]
Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:27610]: In this instance it should be "A" since "urothelial" starts with a consonant sound, like "urine" or "university".

**Pull request merged**